web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -1182,6 +1182,8 @@
     "mapConfigContext-help": "URL to the map context file which can contain the variable lang, for 3 letters language code, between '{' and '}'",
     "mapConfigExtent": "Extent, expressed in current projection",
     "mapConfigLayers": "Layer objects in JSON: {\"type\"=\"XXX\",\"arg1\":\"YYY\",...}",
+    "mapConfigSearchGeodesicExtents": "Use geodesic extents for metadata results",
+    "mapConfigSearchGeodesicExtents-help": "When enabled, use the metadata extent full polygon projected (usually a trapezoid when displaying the map in a local projection). When disabled create a polygon using min /max (displayed as a a rectangle) ",
```

Comment:
I would change this into:
"By default, the displayed metadata extents are planar (i.e. rectangular). If you enable this option, the metadata extents will become geodesic. If the map uses a projected coordinate system, this may lead to non-rectangular extents (e.g. trapezoid)."